dvc/fs/dvc.py

```diff
@@ -151,17 +151,6 @@ def repo_url(self):
             return None
         return self.repo.url
 
-    @property
-    def config(self):
```

Comment:
I thought that this was used for ReferenceODB, but I don't see it being used these days.

Reply:
Yeah, it used to be. Thanks!